bindings/python/tests/test_simple.py

```diff
@@ -59,6 +59,20 @@ def test_accept_path(self):
         os.remove(Path("./out.safetensors"))
 
 
+class WindowsTestCase(unittest.TestCase):
+    def test_get_correctly_dropped(self):
+        tensors = {
+            "a": torch.zeros((2, 2)),
+            "b": torch.zeros((2, 3), dtype=torch.uint8),
+        }
+        save_file_pt(tensors, "./out.safetensors")
+        with safe_open("./out.safetensors", framework="pt") as f:
+            pass
+
+        with open("./out.safetensors", "w") as g:
+            g.write("something")
```

Comment:
I'm not sure I understand how this tests if the file descriptor is correctly dropped.

Can you not have multiple open fds on a file?

Reply:
This is the test that proves it fails before the actual modifications:

https://github.com/huggingface/safetensors/actions/runs/3957656229/jobs/6778277781

The answer is that no, at least not on Windows.